boards/arm/circuitdojo_feather_nrf9160/circuitdojo_feather_nrf9160_common.dts

```diff
@@ -75,6 +75,14 @@
 				<1 0 &gpio0 14 0>,   /* ADC1 = AIN2 */
 				<0 0 &gpio0 13 0>;   /* ADC0 = AIN1 */
 	};
+
+	nrf-ps-en {
```

Comment:
Perhaps it would be good to add a label for this node so that it could be referenced easier.
```suggestion
	nrf_ps_en: nrf-ps-en {
```

Reply:
regulator is currently not used at runtime (it's automatically turned on and left as is forever), this is why I did not add a node label.

Reply:
I know. I just thought that some user might want to disable it at runtime. But maybe that's a contrived scenario.